webapp/src/index.js

```diff
@@ -64,3 +64,14 @@ class PluginClass {
 }
 
 global.window.registerPlugin(pluginId, new PluginClass());
+const getServerRoute = (state) => {
+    const config = getConfig(state);
+    let basePath = '';
+    if (config && config.SiteURL) {
+        basePath = new URL(config.SiteURL).pathname;
+        if (basePath && basePath[basePath.length - 1] === '/') {
+            basePath = basePath.substr(0, basePath.length - 1);
+        }
+    }
+    return basePath;
+};
```

Comment:
Nit pick: add line break

Reply:
@hanzei sure , have resolved the comments 
hope this will work fine

Reply:
The line break is still mising.

Reply:
do we want to have a line break after return statement and closing bracket of function ?

Reply:
The line break should be before the return statement

Reply:
![Screenshot 2021-07-15 at 10 30 07 AM](https://user-images.githubusercontent.com/87431235/125731346-2068e79b-d91e-4671-a50a-abace564801f.png)


i guess the new change is not getting reflected , have attached a screenshot of same

Reply:
The UI also shows that a linebreak is missing.

Reply:
Don't know why it is not reflecting here 
can check this 
https://github.com/mattermost/mattermost-plugin-github/blob/b392c16ddb9ff65b2cef134a798a96581802095c/webapp/src/index.js#L81

Reply:
The file you linked also doesn't have a line break in the last line. See https://raw.githubusercontent.com/mattermost/mattermost-plugin-github/b392c16ddb9ff65b2cef134a798a96581802095c/webapp/src/index.js.

Reply:
you mean there should be a line break before and after return statement ? or line break at last line of the file

Reply:
There should be a line break at the last line of the file. Also, do you mind moving  this `getServerRoute` function to the `selectors.js` file?

Reply:
done